plugins/inputs/cisco_telemetry_mdt/ems/ems.pb.go

```diff
@@ -0,0 +1,1742 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I removed ems since it belonged to dialin (see above).
There is unfortunately no good updated repository here for this purpose. Some carry the latest .proto files but no up-to-date .pb.go.

Reply:
Ideally everything in this repository would be only under the top MIT license, while we can still distribute this code as Apache 2.0 it does cause some confusion and difficulties keeping track of more than one license in the repository.  Could you either license these as MIT (essentially removing this header) or move these pb.go and .proto files to a new github repository?